### ISSUE TITLE
Logout cli

### ISSUE THREAD

**Issue by d1psy-sh** [charmbracelet/inspo#12]
**Is your feature request related to a problem? Please describe.**
I'd like to have a way of logging out / lock screen / shutdown / etc. without memorizing the specific command.

**Describe the solution you'd like**
A TUI program similar to the 'Power Off' option in most desktop environments.

**Include projects that are similar in nature**
I didn't find anything similar in nature so I built it myself [byebye](https://github.com/d1psy-sh/byebye), but I would love to extend the features and have some advice on the code style.

(First time making an issue. How am I doing?)

**Comment by AhmedAlamoudi192** [comment]
Hello @d1psy-sh , I'm a little new to this whole cli thing, but I just wanted to ask about your choice of libraries for this program.

I didn't understand the program at first but then once you shared [byebye](https://github.com/d1psy-sh/byebye) I kinda got the idea and think that you could make the program accomplish the same thing with a simpler library from [Charm](https://github.com/charmbracelet) called [gum](https://github.com/charmbracelet/gum) I think it suits your needs a lot better, tell me what you think about the following script


```bash
#!/bin/bash

OPTION=$(gum choose --limit 3 "poweroff" "reboot" "logout")

if [ $OPTION = "poweroff" ]
then
  poweroff
fi

if [ $OPTION = "reboot" ]
then
  reboot
fi

if [ $OPTION = "logout" ] 
then
  # this is because 'xdg-screensaver lock'
  # did not work for me feel free to change
  # it back

  # xda-screensaver lock
  qtile cmd-obj -o cmd -f shutdown
fi
```

Just install gum then put the previous code in a file, 
make it executable with `chmod +x <path-to-file>` 
and put it somewhere in your PATH

now you can call it anywhere from the terminal in the same way.

**Comment by d1psy-sh** [comment]
@AhmedAlamoudi192 Yes, I am fully on your side with gum. It would be way easier. I wrote this program while starting to learn a bit of Go and I thought it was a nice easy project to use bubbletea on. I also liked the idea that I could style the app the way I wanted to, if I remember correct gum is a bit limited with styling (I didn't look into gum that carefully).

**Comment by AhmedAlamoudi192** [comment]
I completely get you, after all this is why this repo was made, to find more ideas for programs using charm_ in general, since we're in the same boat check out this amazing website I found while looking for more projects to get familiar with go: [gophercises](gophercises.com)
I have been enjoying the projects so far.
